Servers/Administration/OJS.Servers.Administration/Controllers/TestsController.cs

```diff
@@ -398,18 +423,4 @@ private IEnumerable<AutoCrudAdminGridToolbarActionViewModel> GetCustomToolbarAct
             },
         };
     }
-
-    private Expression<Func<Test, bool>> GetMasterGridFilter()
-    {
-        Expression<Func<Test, bool>> filterByLecturerRightsExpression =
-            this.lecturerContestPrivilegesBusinessService.GetTestsUserPrivilegesExpression(
-                this.User.GetId(),
-                this.User.IsAdmin());
-
-        var filter = this.TryGetEntityIdForNumberColumnFilter(ProblemIdKey, out var problemId)
-            ? t => t.ProblemId == problemId
-            : base.MasterGridFilter;
-
-        return filterByLecturerRightsExpression.CombineAndAlso(filter);
```

Comment:
Delete `CombineAndAlso` if not used anymore.

Reply:
Removed.